go/vt/vtgate/engine/ordered_aggregate.go

```diff
@@ -186,6 +186,17 @@ func (oa *OrderedAggregate) GetTableName() string {
 	return oa.Input.GetTableName()
 }
 
+// GetCollations specifies the collation ID value for columns.
+func (oa *OrderedAggregate) GetCollations() map[int]collations.ID {
```

Comment:
this method is only used from this file and it's tests, so I don't think we need to make it public

Reply:
made the change